core/integration/bookkeeping/plugin.go

```diff
@@ -284,10 +330,11 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 		}
 
 		envState := env.CurrentState()
+
 		if envState != "RUNNING" {
-			return updateRunFunc(runNumber64, "bad")
+			return updateRunFunc(runNumber64, "test", time.Now(), time.Time{}, time.Now(), time.Time{})
```

Comment:
What is the significance of "test" in this context?

Reply:
It is the default for the run quality, as per what George told me